plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/cnf/ServerModule.java

```diff
@@ -50,6 +53,8 @@
 	
 	protected ServersView2 view;
 	
+	private String serverModuleName;
```

Comment:
is this variable needed? in implemented methods is used: getLabel().getName()

Reply:
This variable is not needed, I have removed it.